src/database/operations/load/loadProposalBundle.ts

```diff
@@ -1,23 +1,42 @@
 import { loadBundle } from './loadBundle';
-import type { JsonResultSet, Bundle, Proposal } from '../../../types';
+import type {
+	JsonResultSet,
+	Bundle,
+	Proposal,
+	AuthContext,
+} from '../../../types';
 
-export const loadProposalBundle = async (queryParameters: {
-	offset: number;
-	limit: number;
-	search?: string;
-	organizationId?: number;
-	createdBy?: number;
-}): Promise<Bundle<Proposal>> => {
+export const loadProposalBundle = async (
+	queryParameters: {
+		offset: number;
+		limit: number;
+		search?: string;
+		organizationId?: number;
+		createdBy?: number;
+	},
+	authContext?: AuthContext,
```

Comment:
I think this is a much cleaner way to handle authorization context in database calls.

(1) It decouples filters from authorization rules.
(2) It's an optional parameter, so if an auth context is not provided then it won't be used.

Our previous implementation "worked" but would have gotten messy really fast for, well, this exact filter!